images/src/ContentReactor.Images/ContentReactor.Images.Services/ImagesService.cs

```diff
@@ -52,30 +56,32 @@ public ImagesService(IBlobRepository blobRepository, IImageValidatorService imag
             var imageId = Guid.NewGuid().ToString();
 
             // create a blob placeholder (which will not have any contents yet)
-            var blob = BlobRepository.CreatePlaceholderBlob(FullImagesBlobContainerName, userId, imageId);
+            var blob = BlobRepository.CreatePlaceholderBlob(FullImagesBlobContainerName, imageId);
 
             // get a SAS token to allow the client to write the blob
-            var writePolicy = new SharedAccessBlobPolicy
+            BlobSasBuilder blobSasBuilder = new BlobSasBuilder
```

Comment:
nit: The surrounding code is predominantly using `var` and there's no ambiguity here as to the type being instantiated. For consistency, I'd recommend using `var` here.

Reply:
Resolved